oops/oops.go

```diff
@@ -7,8 +7,39 @@ import (
 	"runtime"
 	"strconv"
 	"strings"
+	"sync/atomic"
 )
 
+var (
+
+	// filePrefixesToShortCircuit is a set of prefixes of files which, when encountered when serializing
+	// an oops error, will skip appending the current frame and all subsequent frames in that stack.
+	// It is useful for truncating long call stacks to make stack traces more concise.
+	filePrefixesToShortCircuit atomic.Value // map[string]struct{}{}
+)
+
+func init() {
+	// Make sure we don't panic if we invoke this package and haven't called `SetPrefixesToShortCircuit`.
+	SetPrefixesToShortCircuit(make(map[string]struct{}))
+}
+
+// SetPrefixesToShortCircuit sets `filePrefixesToShortCircuit` to `prefixes`. While it is thread-safe, it is the
+// caller's responsibility to ensure that there are  not multiple callers in the same program calling
+// `SetPrefixesToShortCircuit` concurrently, as this  may cause unexpected behavior where different prefixes are
```

Comment:
Rm double spaces after "are" and "may"

Reply:
I don't see double-spaces after 'are', but I removed them before 'may'.